wishlist/wishlist.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Aniket singh" />
+    <meta name="description" content="wishlist page for PlayX-shop" />
+    <link rel="stylesheet" href="https://playx-ui.netlify.app/styles/main.css" />
+    <link rel="stylesheet" href="wishlist.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+        integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+    <link rel="icon" href="assets/Screenshot 2022-02-12 222538.png" type="image/png" />
+    <title>Document</title>
+</head>
+
+<body>
+    <nav class="nav-conatainer flex  flex-row p-xs">
+        <div class="nav-brand font-xl f-l"><a class="nav-brand-a" href="">PlayX</a></div>
+        <form class="p-s">
+            <input class="nav-input font-xl f-m" type="search" name="search" placeholder="Search..">
+        </form>
+        <ul class="nav-right flex p-s">
+            <li class="pl-m"><button class="nav-login-btn f-s"><a class="" href="">Login</a></button></li>
+            <li class="nav-love-icon pl-l">
+                <a class="nav-loveicon-a" href=""><i class="far fa-heart f-l"></i></a>
+                <span class="nav-number f-s br-s">5</span>
+            </li>
+            <li class="pl-l">
+                <div class="badge">
+                    <a class="nav-bag-icon" href=""><i class="fa fa-shopping-cart badge-icon relative f-l"></i></a>
+                    <span class="badge-greenNumber br-s">10</span>
+                </div>
+            </li>
+        </ul>
+    </nav>
+    <!--main body-->
+    <h1 class="whislist-heading mt-l f-xl font-xl">My Wishlist</h1>
+    <div class="whislist-product flex flex-center mb-l  mt-l">
+        <div class="cards_withBadge">
+            <div class="header">
+                <image class="card-image_one" src="../assets/shopping.webp" alt="controler-image">
+                    <span class="card-icon p-xs br-s"><i class="far fa-heart"></i></span>
+            </div>
+            <div class="body  flex flex-center flex-column">
```

Comment:
give some margin between footer and body

Reply:
will do that